### DISCUSSION TITLE
Automation

### DISCUSSION THREAD

**Discussion by bostonkassidy** [EnviralDesign/GeoPix#41] [Q&A]
I am planning to use GeoPix in a permanent installation (hopefully). I have a few questions.
Is there a way to load directly to a project, without having to load from within Geopix and without a splash screen?
Secondly, is there any plan to import the timer CHOP or something similar. I usually automate my content with timers and logics. I have a backup plan to midi map everything and have ableton in charge of the "show". A future tutorial on midi would be great. 

I love Geopix so far, keep it up!

**Comment by EnviralDesign** [comment]
Hey there,

Awesome to hear about your permanent install - if you want to auto load a specific project file you can set it up via the Tools -> Auto Loader Tool. It updates the preferences JSON file on disk, which is read on boot. 

As for timing, it really depends on what you're trying to do. You can make a timeline based playback system in GP that plays on it's own, and loops for ever. You can make macros and align them on the timeline sequence editor to do that.

Also, if you want you can generate your visuals outside of GP, and stream them into Macros / IO nodes via video capture nodes. 

Currently there's not a way to drive the timeline time externally, though this should be fairly straight forward to implement if you run into that roadblock, let me know.

**Reply by bostonkassidy** [reply]
Thanks a lot for the quick reply. I will def be using the video in feature, very cool.
After thinking about it a bit more, I think the best way to have a dynamic show that will loop endlessly is the midi option instead of my usual methods of timers. 
Can you assign a midi note to macro buttons in the perform tab? I would be cool to have it setup with an interface so it could be jammed on live, or set on an "autopilot" with a midi sequence coming from ableton.

**Reply by EnviralDesign** [reply]
yes you can! and that sounds like a neat idea, having a backup autopilot.

I haven't made a dedicated midi mapping tutorial yet, but you can find the relevant details here:
https://github.com/EnviralDesign/GeoPix/wiki/Building-a-multi-group-VJ-System

**Reply by bostonkassidy** [reply]
excellent! ill post some pics of the progress in the show off section :)

Thanks a bunch
